backend/web/workflows/sft_fixer.py

```
@@ -0,0 +1,969 @@
+            message=f"Fix '{fix_request.fix_name}' {'completed successfully' if success else 'failed'}",
+        )
+
+    except NotImplementedError as e:
+        return ExecuteScriptResponse(
+            success=False,
+            instance_id=fix_request.instance_id,
+            status="not_implemented",
+            output=None,
+            error=str(e),
+            exit_code=None,
+            message=f"Fix execution not yet implemented for this provider",
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)